examples/end-to-end-session-based/03-Session-based-Yoochoose-multigpu-training-PyT.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>In the following sentence, maybe worth adding that in the context of RecSys, the larger number of parameters sits on embedding tables</p><ul><li>Model Parallel: If the model is too large to fit on a single GPU, the parameters are distributed over multiple GPUs</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/521/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>